src/commands/checkInv.js

```diff
@@ -1,9 +1,9 @@
-export const checkInv = (inventory) => {
-  if (inventory.length === 0) {
+export const checkInv = (ctx) => {
+  if (ctx.inventory.length === 0) {
```

Comment:
This is how you can avoid writing `ctx.` everywhere - called "destructuring" if you didn't notice. I did this already for the `process.env` part.

```suggestion
export const checkInv = ({ inventory }) => {
  if (inventory.length === 0) {
```

You kind of just build a thing that looks kind of like the object you expect, then it will extract those things into the variables for you. Pretty handy.

Works in variable assignments/declarations, and in function parameters:
```javascript
const { x } = { x: 1, y: 2 };
console.log(x); // 1

const [a, b, c] = [1, 2, 3, 4];
console.log(a, b, c); // 1 2 3

function test({ x, y }) {
  return a + b;
}

console.log(test({ x: 1, y: 2, z: 15 })); // 3
```